content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,206 @@
+  as xml files and you then have to upload them on the Agenzia dell'Entrate website.
+- :guilabel:`Test` mode sends invoices to a non-production service. THerefore a communication is
+  established between Odoo and the test server of the Public Administration and only send invoices
+  to a selected few `fake` destinations.
+- :guilabel:`Official` is a production mode that sends your invoices directly to the Agenzia
+  dell'Entrate.
 
-Open your sales journal. Under the *Advanced Settings* tab, in the **Electronic invoicing** field,
-check *FatturaPA* and *Save*.
+.. warning::
+   The choice of the mode is **not reversible**. Once in :guilabel:`Official` mode it is not
+   possible to get back to the :guilabel:`Test` and :guilabel:`Demo`. Equally is not possible to go
+   from :guilabel:`Test` to :guilabel:`Official`.
 
-.. image:: italy/journal.png
-   :align: center
-   :alt: FatturaPA feature in the sales journal's settings
+After having selected the right mode, you need to accept the terms and conditions right below the
+mode selection. Once you have :guilabel:`Saved` the mode you want, you can start recording your
+Accounting transactions in Odoo.
 
 Issue invoices
---------------
+==============
 
-Send an invoice the way you would do it normally. The EDI process proceeds automatically.
+The **FatturaPA** feature is automatically active, you can check it in
+:menuselection:`Configuration --> Journals --> Customer Invoices -->  Advanced Settings`.
+You can create a new invoice by going to :menuselection:`Accounting dashboard --> New invoice`, once
+confirmed the e-invoice will be generated and sent.
 
-.. image:: italy/sending.png
-   :align: center
-   :alt: Asynchronous sending on a customer invoice
+.. warning::
+   The e-invoice is only sent to the government if you are in :guilabel:`Official` mode.
 
-You can check the current status of your customer invoice under the **Electronic invoicing** field.
+You can check the current status of your customer invoice under the :guilabel:`Electronic invoicing`
+field. The xml file is accessible in the chatter of the invoice.
 
 .. image:: italy/processing.png
    :align: center
    :alt: Electronic invoicing status (waiting for confirmation)
 
+Vendor bills
+============
+
+As for invoices, when creating a vendor bill, the :guilabel:`FatturaPA` option is automatically
+activated in the Vendor bill Journals' advanced settings.
+When inserting taxes in a vendor bill, you have the possibility to choose **Reverse Charge** taxes,
+these are automatically activated in the Italian fiscal position. By going to
+:menuselection:`Configuration --> Taxes`, you will see that the 10% and 22% services and goods taxes
+are activated and have the correct tax grids; these are set up automatically to ensure the correct
+booking of accounting entries and display of the tax report.
+Once the bill is confirmed, a banner appears, and the bill can be sent to the EDI service.
```

Comment:
The FatturaPA option is automatically activated when the :guilabel:`Italy - Accounting` module is installed. You can check it by going to :menuselection:`Accounting --> Configuration --> Journals --> Vendor Bills --> Advanced Settings`. 

When inserting taxes in a vendor bill, you can choose :guilabel:`Reverse Charge` taxes; these are automatically activated in the Italian fiscal position. 

By going to :menuselection:`Configuration --> Taxes`, you see that the 10% and 22% services and goods taxes are activated and have the correct tax grids; these are set up automatically to ensure the correct booking of accounting entries and display of the tax report. 

Once the bill is confirmed, a banner appears, and the bill is sent in electronic format.
```

Reply:
I didn't see the Reverse Charge... Maybe it would be good to clarify what it is?